vita3k/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -103,8 +105,16 @@ EXPORT(int, sceAppUtilDrmClose) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppUtilDrmOpen) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceAppUtilDrmOpen, emu::SceAppUtilDrmAddcontId *dirName, SceAppUtilMountPoint *mountPoint) {
+    const auto drm_content_id_path{ fs::path(host.pref_path) / (+VitaIoDevice::ux0)._to_string() / host.io.device_paths.addcont0 / reinterpret_cast<char*>(dirName->data) };;
```

Comment:
```suggestion
    const auto drm_content_id_path{ fs::path(host.pref_path) / (+VitaIoDevice::ux0)._to_string() / host.io.device_paths.addcont0 / reinterpret_cast<char*>(dirName->data) };
```
also, you ran the format script right?

Reply:
Have est format alone normally is good